composer.json

```diff
@@ -40,8 +42,24 @@
             "DotTest\\AnnotatedServices\\": "tests"
         }
     },
+    "scripts": {
+        "check": [
+            "@cs-check",
+            "@test"
+        ],
+        "cs-check": "phpcs",
+        "cs-fix": "phpcbf",
+        "test": "phpunit --colors=always",
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml",
+        "static-analysis": "psalm --shepherd --stats"
+    },
     "suggest": {
         "doctrine/cache": "To cache the result of processing the annotations and speed-up your application.",
         "doctrine/orm": "To be able to inject doctrine repositories."
+    },
+    "config": {
```

Comment:
Not a requirement, rather a nice-to-have, but for consistency across all our packages, let's add under the `config` section:
`"sort-packages": true,`